src/_gettsim/transfers/wohngeld.py

```diff
@@ -159,8 +159,8 @@ def wohngeld_eink_vor_freib_m_mit_elterngeld(  # noqa: PLR0913
     arbeitsl_geld_m: float,
     sonstig_eink_m: float,
     eink_rente_zu_verst_m: float,
-    kind_unterh_erhalt_m: float,
-    unterhaltsvors_m: float,
+    # kind_unterh_erhalt_m: float,
+    # unterhaltsvors_m: float,
```

Comment:
Remove or comment in again? Same below and in do string.

Reply:
I’ll let @ChristianZimpelmann judge this one. As far as I can see #357 is not resolved and Christian only has a vague feeling about what is correct? In any case we should mark it as a todo (will do that later).

Reply:
I reverted the changes. Since #739 is fixed, both versions lead to the same result unless a child receives both alimony payments and earnings from a job. Still unsure what is correct in that case.